aspnetcore/host-and-deploy/iis/development-time-iis-support.md

```diff
@@ -1,102 +1,108 @@
 
-![Modifying Visual Studio features: The Workloads tab is selected. In the Web and Cloud section, the ASP.NET and web development panel is selected. On the right in the Optional area of the Summary panel, there's a check box for Development time IIS support.](development-time-iis-support/_static/development_time_support.png)
+   The component is listed in the **Optional** section under **Development time IIS support** in the **Installation details** panel to the right of the workloads. The component installs the [ASP.NET Core Module](xref:host-and-deploy/aspnet-core-module), which is a native IIS module required to run ASP.NET Core apps with IIS.
 
 ## Configure the project
 
 ### HTTPS redirection
 
-For a new project, select the check box to **Configure for HTTPS** in the **New ASP.NET Core Web Application** window:
+For a new project that requires HTTPS, select the check box to **Configure for HTTPS** in the **Create a new ASP.NET Core Web Application** window. Selecting the check box adds [HTTPS Redirection and HSTS Middleware](xref:security/enforcing-ssl) to the app when it's created.
 
-![New ASP.NET Core Web Application window with the Configure for HTTPS check box selected.](development-time-iis-support/_static/new-app.png)
+For an existing project that requires HTTPS, use HTTPS Redirection and HSTS Middleware in `Startup.Configure`. For more information, see <xref:security/enforcing-ssl>.
 
-In an existing project, use HTTPS Redirection Middleware in `Startup.Configure` by calling the [UseHttpsRedirection](/dotnet/api/microsoft.aspnetcore.builder.httpspolicybuilderextensions.usehttpsredirection) extension method:
+For a project that uses HTTP, [HTTPS Redirection and HSTS Middleware](xref:security/enforcing-ssl) aren't added to the app. No app configuration is required.
 
-```csharp
-public void Configure(IApplicationBuilder app, IHostingEnvironment env)
-{
-    if (env.IsDevelopment())
-    {
-        app.UseDeveloperExceptionPage();
-    }
-    else
-    {
-        app.UseExceptionHandler("/Error");
-        app.UseHsts();
-    }
+### IIS launch profile
 
-    app.UseHttpsRedirection();
-    app.UseStaticFiles();
-    app.UseCookiePolicy();
+Create a new launch profile to add development-time IIS support:
 
-    app.UseMvc();
-}
-```
+::: moniker range=">= aspnetcore-3.0"
 
-### IIS launch profile
+1. Right-click the project in **Solution Explorer**. Select **Properties**. Open the **Debug** tab.
+1. For **Profile**, select the **New** button. Name the profile "IIS" in the popup window. Select **OK** to create the profile.
+1. For the **Launch** setting, select **IIS** from the list.
+1. Select the check box for **Launch browser** and provide the endpoint URL.
 
-Create a new launch profile to add development-time IIS support:
+   When the app requires HTTPS, use an endpoint with the HTTPS protocol (`https://`). When the app uses HTTP, use an endpoint with the HTTP protocol (`http://`).
```

Comment:
When the app requires HTTPS, use an HTTPS endpoint (`https://`). For HTTP, use an HTTP (`http://`) endpoint .